src/src/presentation/dashboard/issuers/IssuersScreen.tsx

```diff
@@ -58,67 +65,144 @@ const IssuersScreen = class IssuersScreen extends NavigationEnabledComponent<
         this.props.getAllIssuerData(limit, page);
     }
 
+    toggleModal() {
+        this.setState({ modalVisible: !this.state.modalVisible});
```

Comment:
Cuando se va a cambiar el estado en base a un estado anterior, hay que usar **setState(state => { ....})**. Esto es porque al momento de cambiar el estado no se aseguro que **this.state.modalVisible** tenga el mismo valor que cuando se llamó.  
Este problema es más evidente en dispositivos de gama baja, con son los del user objetivo de la app.